### PR TITLE
Update prerequisites.md

### PR DESCRIPTION
**Pull request by msbemba**
removed - Enterprise Mobility + Security E3 from Azure AAD p2

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2783

File: microsoft-365/managed-desktop/get-ready/prerequisites.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Microsoft Managed Desktop requires certain license options in order to function.
 
 |License option |Available in *any* of these license products |How Microsoft Managed Desktop uses it|
 |-------------|-------------|-------------|
-|Azure Active Directory Premium P2     |- Microsoft 365 E5<br>- Microsoft 365 E3 + Microsoft 365 *E5* Security Add-on<br>- Enterprise Mobility + Security E5<br>- Enterprise Mobility + Security E3<br>- Azure Active Directory Premium P2|  Provides access to Microsoft Cloud Services; allows AutoPilot to register devices      |
+|Azure Active Directory Premium P2     |- Microsoft 365 E5<br>- Microsoft 365 E3 + Microsoft 365 *E5* Security Add-on<br>- Enterprise Mobility + Security E5<br><br>- Azure Active Directory Premium P2|  Provides access to Microsoft Cloud Services; allows AutoPilot to register devices      |
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/prerequisites.md]:
```suggestion
|Azure Active Directory Premium P2     |- Microsoft 365 E5<br>- Microsoft 365 E3 + Microsoft 365 *E5* Security Add-on<br>- Enterprise Mobility + Security E5<br><br>- Azure Active Directory Premium P2|  Provides access to Microsoft cloud services; allows Autopilot to register devices      |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by jaimeo** [comment]
@siavoi can you confirm this edit is accurate?

**Comment by jaimeo** [comment]
Per the SME, this edit is not valid. SME is providing a comprehensive update to this article shortly.

**Comment by msbemba** [comment]
Thank you, Jaimeo. Will wait for the update from SME.

**Pull request closed without merging**